src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -560,7 +560,15 @@ public bool Equals(LayoutBuilder other)
 
 			public void Build()
 			{
-				MeasuredSize = UpdateLayout(AvailableSize, _exactWidth);
+				if (AvailableSize.Width < 0 || AvailableSize.Height < 0)
+				{
+					// The layout is invalid, we can't build it.
+					MeasuredSize = new Size(0, 0);
+				}
```

Comment:
That's somewhat curious.

Reply:
Yes, most likely due to the fact that the ComboBox is keeping the text displayed but zero sized in some cases. So far it didn't cause any problems, but will not it to check